docs/topics/native/native-cocoapods-dsl-reference.md

```diff
@@ -122,12 +122,12 @@ a separate function call.
-| `source`      | Configures the Pod from: <list><ul><li>The Git repository using `git()`. In the block after `git()`, you can specify `commit` to use a specific commit, `tag` to use a specific tag, and `branch` to use a specific branch from the repository</li><li>The local repository using `path()`</li><li>An archived (tar, jar, zip) Pod folder using `url()`</li></ul></list> |
-| `packageName` | Specifies the package name.                                                                                                                                                                                                                                                                                                                                              |
-| `extraOpts`   | Specifies the list of options for a Pod library. For example, specific flags: <code style="block" lang="Kotlin">extraOpts = listOf("-compiler-option")</code>.                                                                                                                                                                                                           |
+| **Name**      | **Description**                                                                                                                                                                                                                                                                                                                                             | 
+|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `version`     | The library version. To use the latest version of the library, omit the parameter.                                                                                                                                                                                                                                                                          |
+| `source`      | Configures the Pod from: <ul><li>The Git repository using `git()`. In the block after `git()`, you can specify `commit` to use a specific commit, `tag` to use a specific tag, and `branch` to use a specific branch from the repository</li><li>The local repository using `path()`</li><li>An archived (tar, jar, zip) Pod folder using `url()`</li></ul> |
```

Comment:
I'll replace <ul> with <list> in some other branch and commit due to the Slack thread I mentioned in our team chat.
Here please pay attention only to the lost "Targets list:".

Reply:
JFYI, the replacement was done [here](https://github.com/JetBrains/kotlin-web-site/pull/3356/files).